
syntax = "proto2";

package Protocal.Respond;

import "RawData/InternalData.proto";

message Respond_LoginAuth {
	required string token = 1;
	required int32 sceneId = 2;
	required Protocal.RawData.PlayerInfo player = 3;
	repeated Protocal.RawData.PlayerInfo neighborhood = 4;
}


message Respond_Logout
{
	required int32 pid = 1;
}

message Respond_Moving {
	repeated Protocal.RawData.RespMovement movementList = 1;
}

message Respond_PlayerAppeared {
	repeated Protocal.RawData.PlayerInfo neighborhood = 1;
}

message Respond_PlayerDisappeared {
	repeated int32 playerIds = 1;
}

message Respond_Rotated {
	repeated Protocal.RawData.RespRotated rotations = 1;
}



